ice-cream.js

```diff
@@ -0,0 +1,13 @@
+function order(number, flavor) {
+  let addS = 'scoop';
+  if (number > 1){
+    addS = 'scoops'
+  }
+  console.log(`Lemme get ${number} ${addS} of ${flavor}`)
+}
+//ex
+order(2, 'vanilla');
```

Comment:
👍 
You could extend it by making a separate function to pluralize words and then call that within your `order` function. That way if you had a another function, say something like `getRecentOrders`, you could reuse the pluralize in both places:
```suggestion
const pluralize = (count, noun, suffix = 's') =>
  `${noun}${count !== 1 ? suffix : ''}`;
  
function order(number, flavor) {
  console.log(`Lemme get ${number} ${pluralize(number, 'scoop')} of ${flavor}`)
}
//ex
order(2, 'vanilla');
```